src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/register_app_interface_request_test.cc

```diff
@@ -460,12 +460,6 @@ TEST_F(RegisterAppInterfaceRequestTest, Run_MinimalData_SUCCESS) {
       .WillByDefault(
           Return(application_manager::HmiInterfaces::HMI_INTERFACE_Buttons));
 
-  EXPECT_CALL(
-      mock_rpc_service_,
-      ManageHMICommand(
-          HMIResultCodeIs(hmi_apis::FunctionID::Buttons_SubscribeButton), _))
-      .WillOnce(Return(true));
-
```

Comment:
@VadymLuchko I would suggest not remove this expect_call but add another one that returns `true` for `CheckHMICapabilities`

Reply:
good idea
fixed in 404dd1fcfea4f3afe0fc51a14c561381a7cc8dd2